protocol/0057-TRAN-transfers.md

```diff
@@ -67,6 +67,9 @@ A recurring transfers needs to contain this specific information:
 - start epoch: at the end of this epoch the first recurring transfer will be made between
 - end epoch (optional): at the end of this epoch the last recurring transfer will be made between, optional. If not specified the transfer run until cancelled (by its creator or by the network as described below).
 - factor, decimal > 0.0 (a factor used with the amount specified for the transfer).
+- transfer interval: number of epochs between transfers, i.e. when 4, funds will be transferred every 4 epochs with the first transfer occurring 4 epochs after the transaction is processed. Must be an integer strictly greater than `0` and less than `100` (this ceiling is for performance reasons and matches the limit on the `window_length` field).
```

Comment:
it would be a lot easier (or at all feasible) if transfer_interval is part of the dispatch strategy. why? the reward has no access (and cannot have access) to which transfer funded the reward account, because there could be multiple transfers with the same dispatch strategy funding the account, so say we have transfer A and transfer B funding a reward account with the exact same dispatch strategy but different interval, the reward engine cannot enforce the cap as it doesn't have information about which look-back period it should use. So while it's not super clean, putting it in the dispatch strategy would work better I think.

Reply:
Yep sounds sensible, agree it will be clearer in the dispatch strategy regrading multiple transfers funding the same reward account.